core/test/config/config-test.js

```diff
@@ -432,7 +432,7 @@ describe('Fraggle Rock Config', () => {
     };
 
     // https://github.com/facebook/jest/issues/11438
-    // expect(initializeConfig(extensionConfig, {gatherMode: 'navigation'}))
+    // await expect(initializeConfig(extensionConfig, {gatherMode: 'navigation'}))
     //   .rejects.toThrow(/did not support any gather modes/);
```

Comment:
This issue is with jest. Does it work with mocha now?

Reply:
sure does